<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>etbarchivewindow</class>
 <widget class="QMainWindow" name="etbarchivewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Date">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>730</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1024</width>
      <height>686</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>984</width>
        <height>630</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>28</y>
        <width>944</width>
        <height>480</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>300</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_play">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>528</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/play-button.png</normaloff>:/new/icons/play-button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_upload">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>528</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>UPLOAD</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/upload.png</normaloff>:/new/icons/upload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rename">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>528</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>RENAME</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/edit.png</normaloff>:/new/icons/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>521</x>
        <y>528</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/bin.png</normaloff>:/new/icons/bin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_copy_move">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>530</y>
        <width>303</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>COPY / MOVE
TO USB </string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/usb-drive.png</normaloff>:/new/icons/usb-drive.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>532</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>944</width>
        <height>480</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_info_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>984</width>
        <height>630</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_copy">
      <property name="geometry">
       <rect>
        <x>543</x>
        <y>530</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>COPY</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/copy.png</normaloff>:/new/icons/copy.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_move">
      <property name="geometry">
       <rect>
        <x>693</x>
        <y>530</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>MOVE</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/send.png</normaloff>:/new/icons/send.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete_2">
      <property name="geometry">
       <rect>
        <x>843</x>
        <y>529</y>
        <width>141</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/bin.png</normaloff>:/new/icons/bin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <zorder>label_info_2</zorder>
     <zorder>treeView_2</zorder>
     <zorder>pushButton_copy</zorder>
     <zorder>pushButton_move</zorder>
     <zorder>pushButton_delete_2</zorder>
     <zorder>label_status</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_button">
    <property name="geometry">
     <rect>
      <x>834</x>
      <y>0</y>
      <width>190</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Time">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>730</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_home_button">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/icons/home-button.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_heading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> ETB ARCHIVES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>934</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/icons/back-arrow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_Date</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>label_button</zorder>
   <zorder>label_Time</zorder>
   <zorder>label_heading</zorder>
   <zorder>pushButton_return</zorder>
   <zorder>pushButton_home_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
